### PR TITLE
feat: add cookie consent banner

### LINKED ISSUE #14: add cookie banner

### PR DESCRIPTION
**Pull request by hasanagh**
adds cookie consent banner

closes #14

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -81,9 +81,12 @@ module.exports = {
     `gatsby-plugin-smoothscroll`,
     `react-scrollspy`,
     {
-      resolve: `gatsby-plugin-google-analytics`,
+      resolve: `gatsby-plugin-gdpr-cookies`,
```
Comment (juancarlosfarah) [inline gatsby-config.js:84]: Why the change?

**Review by juancarlosfarah — APPROVED** [review]
@hasanagh, thanks for this. Just a quick question in one of the comments.

File: gatsby-config.js (hunk 1 above)
Comment (hasanagh) [reply gatsby-config.js:84]: @juancarlosfarah  Basically, the new plugin only starts tracking using GA after the user presses "ok" in the cookie consent. This is for GDPR compliance.

Comment (juancarlosfarah) [reply gatsby-config.js:84]: Great. You can go ahead and merge.
